src/app/shared/data-structure/data-structure.component.html

```diff
@@ -0,0 +1,25 @@
+<div fxLayout="row" fxLayoutAlign="space-between start">
+  <mat-sidenav-container fxFLex layout-padding layout-align="start" class="sidenav-container">
+    <mat-sidenav fxFlex mode="side" position="end" #sidenav class="sidenav">
+      <div fxLayout="column">
+        <div fxLayout="row" fxLayoutAlign="start center" layout-margin>
+          <div fxFlex fxLayout="row" fxLayoutAlign="start center">
+            <mat-icon layout-margin>tune</mat-icon>
+            <span> {{ compName + '.CONFIGURATION' | oTranslate }}</span>
+          </div>
+          <div fxLayoutAlign="end center" fxFlex>
+            <button mat-button (click)="toggle()">
```

Comment:
```suggestion
            <mat-icon click)="toggle()">close</mat-icon>
```

I think this is better to avoid the hover on the button